### PR TITLE
Improve manual.md

### PR DESCRIPTION
**Pull request by felixboehm**
@IljaN @micbar 
Can you review, and add missing tickets and links in the text, please.

### CODE REVIEW

File: docs/manual.md [hunk 1]
```diff
@@ -1,66 +1,71 @@
...
 
-Finally delete the user on the old instance (This can not be undone!):
+Finally you can delete user1 on the **source instance** (This can not be undone!):
 
 ``$ ./occ user:delete user1``
 
-Note: If the user is stored in the database you need to manually reset his password (see Limitations)
 
-##What is exported?
+## What is exported and imported?
```
Comment (mmattel) [inline docs/manual.md]: Pls use Title Case, suggestion: `What is Exported, what is Imported

**Comment by mmattel** [comment]
There are missing ticket links `(Ticket: Link?)`, fix that before merging or keep it and fix it later?

**Comment by mmattel** [comment]
If there are changes, pls open a doc ticket too, as we already have a documentation page, see https://doc.owncloud.org/server/admin_manual/maintenance/export_import_instance_data.html
It does not matter if the changes addressed are under development, but atm, if I am not looking at, this gets lost. We can always adopt or close a doc ticket, but having one will help to follow up.

@micbar fyi

**Comment by IljaN** [comment]
@mmattel / @settermjd I know that the current docs process is to open a seperate PR in the documentation repository. But I personally would strongly prefer to have the canonical version of the documentation as close as possible to the source-code (docs folder in this repo). With this structure it would be possible to update docs and code simultaneously in one PR. I heard that it is possible with the new docs systems to collect the documentation parts from different repos.

On the other hand I don't want to create a special-snowflake process just for data-exporter.

**Comment by mmattel** [comment]
@IljaN possible, but **very** bad for this repro. The view repros where we do this have a docs folder, but the content is written in Antora and follows the document build rules which is a lot of pre-work. 
Consider this repro docs as temproary solution of a time where DE was in his "early pregnant" phase. We have already created the docs as linked above and we should take that one as the major and only one. Anything else turns over the longer run into a managing and content nightmare.
I have a full understanding about your concerns but please be assured that you have full docs support keeping the content up to date - your contribution is very welcomed.

**Pull request merged**